ngraph/python/tests/conftest.py

```diff
@@ -73,3 +73,7 @@ def pytest_collection_modifyitems(config, items):
         skip_this_backend = keywords[backend_name]
         if skip_this_backend in item.keywords:
             item.add_marker(skip_markers[backend_name])
+
+        if item.name.startswith(("test_reduce_operation[None-False",
```

Comment:
I don't think we should use `conftest` for this. Can we apply this decorator where the test is defined?

Reply:
test definition is a generator of almost 100 TestCases, only few xfails, I don't know another way to set only particular constellation of TestCases.

Reply:
Anyway, I split definition and select cases as xfail instead of adding this logic into conftest